src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
+It' possible to define some attributes to the properties as:
 
-### PartialEq
+- optional and initialized with Rust default value
+- optional and initialieed with component default value
+- mandatory, the parent must define a value for the attribute
 
-It is likely to make sense to derive `PartialEq` on your props if you can do this. Using `PartialEq` makes it much easier to avoid unnecessary rerendering \(this is explained in the **Optimizations & Best Practices** section\).
+> remark: for convenience and/or performance optimization `Props` can derive `PartialEq` to avoid to re-render if there is no changed.cf. `change` method
+
+```rust
+use yew::prelude::*;
+
+pub struct UseOfPropertyComponent {
+    link: ComponentLink<Self>,
+    props: Props,
+    name: String,
+    show_message: bool,
+}
+
+pub enum Msg {
+    Click(),
+}
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props{
+    pub name: String,
+}
+
+impl Component for UseOfPropertyComponent {
+    type Message = Msg;
+    type Properties = Props;
+
+    fn create(props: Self::Properties, link: ComponentLink<Self>) -> Self {
+        Self {
+            link,
+            props: props.clone(),
+            name: props.name.into(),
+            show_message: false,
+        }
+    }
+
+    fn update(&mut self, msg: Self::Message) -> ShouldRender {
+        match msg {
+            Msg::Click() => self.show_message = true,
+        }
+        true
+    }
+
+    fn change(&mut self, props: Self::Properties) -> ShouldRender {
+        if self.props != props {
+            self.props = props;
+            true
+        } else {
+            false
+        }
+    }
+
+    fn view(&self) -> Html {
+        if !self.show_message {
+            html! {
+                <>
+                    <button onclick=self.link.callback( |_| Msg::Click() )>{"Click here!"}</button>
+                </>
+            }
+        } else {
+            html! {
+                <>
+                    <h1>{format!("Hello {}",self.name)}</h1>
+                </>
+            }
+        }
+    }
+}
+
+```
+
+In order to use this component you have to:
+
+```html
+// ...
+    <div class="full-height">
+        {"In this example we pass the name as parameter of the Yew component."}
+        <UseOfPropertyComponent name="Clark"/>
+    </div>
+// ...
+
+```
+
+## Define the properties structure
```

Comment:
"Define" => "Defining"
"structure" => "struct"

Reply:
I am not a fan of "...ing" ... I found it "heavier"
Anyway -> I apply your recommandation

I will be pleased to revert this modification ;-)